doc/content/source/postprocessors/MultiPeriodAverager.md

```diff
@@ -0,0 +1,29 @@
+# MultiPeriodAverager
+
+The Postprocessor will take the average of the `getPostprocessorValueOld("value")` when a new period begins and then average the value after a specified number of periods. Until the number of periods over which you are averaging has passed the value of the Postprocessor will reflect the previously calculated average.
+
+!alert warning
+Before the first average is calculated the value of this post processor will be 0.
+
+
+!alert note title=Proper Usage Hint
+This Postprocessor is intended to be used with the [PeriodicTimeIntegratedPostprocessor.md].
+
+
+## Example Multi Period Mean
+
+!media media/multi_period_mean.png
+       style=width:50%;display:block;margin-top:3em;margin-left:auto;margin-right:auto;
+
+The results in the figure above are from `test/tests/postprocessors/periodic_amplitude_regulator/periodic_modifier.i` with a more refined time step, `dt = 1e-2` with `number_of_periods = 2`. The figure above shows the expected behaviour of this Postprocessor.
```

Comment:
Marking this unresolved just because there is now an empty `Example Multi Period Mean` section in the documentation. I recommend to either remove this section or add an example that doesn't relay on knowledge of other postprocessors (if it can be helped).